hedera-node/src/main/java/com/hedera/services/records/TransactionRecordService.java

```diff
@@ -88,6 +102,14 @@ public void externaliseEvmCallTransaction(final TransactionProcessingResult resu
 		txnCtx.setStatus(getStatus(result, SUCCESS));
 		txnCtx.setCallResult(EvmFnResult.fromCall(result));
 		txnCtx.addNonThresholdFeeChargedToPayer(result.getGasPrice() * (result.getGasUsed() - result.getSbhRefund()));
```

Comment:
We can now rename this method `addFeeChargedToPayer()`; threshold fees have not existed for a long time.

Reply:
Renamed